tests/daemon/unittest_service_db.cc

```diff
@@ -19,25 +19,23 @@
 TEST (serviceDB, set_pipeline_n)
 {
   MLServiceDB &db = MLServiceDB::getInstance ();
-  int gotException = 0;
 
   db.connectDB ();
+
   try {
     db.set_pipeline ("", "videotestsrc ! fakesink");
+    FAIL ();
   } catch (const std::exception &e) {
-    ml_loge ("Got Exception: %s", e.what ());
-    gotException = 1;
+    /* expected */
```

Comment:
I am confused a little bit here. ```db.set_pipeline()``` will throw an exception because of the empty key. Then, ```FAIL()``` is just a guard like ```g_return_if_reached()```?

Reply:
Right, if no exception occurs with empty key, it reports tc failure.